tracer/build/supported_versions.json

```diff
@@ -259,9 +259,9 @@
         "minVersionAvailableInclusive": "3.1.0",
         "minVersionSupportedInclusive": "3.1.0",
         "minVersionTestedInclusive": "3.1.5.3",
-        "maxVersionSupportedInclusive": "3.7.407.1",
-        "maxVersionAvailableInclusive": "3.7.407.1",
-        "maxVersionTestedInclusive": "3.7.407.1"
+        "maxVersionSupportedInclusive": "3.7.407.4",
```

Comment:
Just curious why we are not changing `maxVersionSupportedInclusive` as 4?

Reply:
Hmmm, this may be an artifact of how I generated the file (it's aut generated). I'll run a full update in a subsequent PR and it should fix all these weird little things 🙂

Reply:
FYI I ran the `GeneratePackageVersions` build task locally and it does update the `maxVersionSupportedInclusive` to 4.0.0.2